Client/Extensions/UIView+Extension.swift

```diff
@@ -51,19 +51,20 @@ extension UIView {
     /// using the desired blur style.
     ///
     /// - Parameter style: The strength of the blur desired
-    func addBlurEffectWithClearBackgroundAndClipping(using style: UIBlurEffect.Style) {
-        guard !UIAccessibility.isReduceTransparencyEnabled, shouldAddBlur else { return }
+    func addBlurEffectWithClearBackgroundAndClipping(using style: UIBlurEffect.Style)
+        -> UIVisualEffectView? {
+        guard !UIAccessibility.isReduceTransparencyEnabled, shouldAddBlur else { return nil }
```

Comment:
```suggestion
    func addBlurEffectWithClearBackgroundAndClipping(using style: UIBlurEffect.Style) -> UIVisualEffectView? {
        guard !UIAccessibility.isReduceTransparencyEnabled, shouldAddBlur else { return nil }
```
nit: we shouldn't have **just** the return value on a single line by itself. We should either have it on the same line OR we should include the argument ending bracket on that line as well (ie `) -> UIVisualEffectView? {`